### PR TITLE
added styles for list

### PR DESCRIPTION
**Pull request by raman1999**
image
image

### CODE REVIEW

File: components/list.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  margin: 1rem 0.2rem;
+  border: 1px solid var(--txtPrimary);
+}
+
+.list-item:hover {
+  background-color: var(--txtPrimary);
+  color: var(--txtSecondary);
+  cursor: pointer;
+}
+/* For normal list items */
```
Comment (ImrozNezami) [inline components/list.css:20]: remove comment

File: components/list.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+.list-normal > li {
+  padding: 4px;
+  margin: 0;
+}
+/* For inline list items */
```
Comment (ImrozNezami) [inline components/list.css:25]: remove comment

File: components/list.css [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+.list-inline > li {
+  display: inline;
+}
+
+/* For stack list items  */
```
Comment (ImrozNezami) [inline components/list.css:30]: remove comment

Comment (raman1999) [reply components/list.css:30]: Thanks for reviewing ,  actually I have added comments for making it easier for others to read code  .

**Review by shuntzhere — APPROVED** [review]
Looks pretty neat to me. The classes used are clear, the styling is clean as well. Good Work.

**Pull request merged**